src/state/apis/limit-orders/limitOrderApi.ts

```diff
@@ -211,16 +174,48 @@ export const limitOrderApi = createApi({
         }
       },
     }),
-    cancelLimitOrders: build.mutation<boolean, { payload: OrderCancellation; chainId: ChainId }>({
-      queryFn: async ({ payload, chainId }) => {
+    cancelLimitOrders: build.mutation<
```

Comment:
This will always cancel a single order only
```suggestion
    cancelLimitOrder: build.mutation<
```

Reply:
2151427d82837a2c91febcaed514dcce0fdc662d